pkg/query-service/model/v3/v3.go

```diff
@@ -415,20 +415,20 @@ func (f *FilterSet) Validate() error {
 }
 
 type FilterItem struct {
-	Key      string      `json:"key"`
-	Value    interface{} `json:"value"`
-	Operator string      `json:"op"`
+	Key      AttributeKey `json:"key"`
+	Value    interface{}  `json:"value"`
+	Operator string       `json:"op"`
 }
 
 type OrderBy struct {
-	ColumnName string `json:"columnName"`
-	Order      string `json:"order"`
+	ColumnName AttributeKey `json:"columnName"`
+	Order      string       `json:"order"`
 }
 
 type Having struct {
-	ColumnName string      `json:"columnName"`
-	Operator   string      `json:"operator"`
-	Value      interface{} `json:"value"`
+	ColumnName AttributeKey `json:"columnName"`
```

Comment:
Same question

Reply:
The Having clause here is a set of conditions on the aggregation result, such as `MIN(span.duration)` > 100, `SUM(span.duration)` <= 1000 or something similar. What is the rationale for making this `AttributeKey`? It doesn't make sense to me. This is not some existing column of (resource)attributes the user is working with. It is on the aggregation result.

Reply:
Understood, I will remove AttributeKey as a requirement for Having